lib/client.js

```diff
@@ -147,6 +147,24 @@ Freshdesk.prototype = Object.create({
 		makeRequest('GET', this._auth, `${this.baseUrl}/api/v2/contact_fields`, null, null, cb)
 	},
 
+	//Agents
+
+	getAgent(id, cb) {
+		makeRequest('GET', this._auth, `${this.baseUrl}/api/v2/agents/${id}`, null, null, cb)
+	},
+
+	listAllAgents(params, cb) {
+		makeRequest('GET', this._auth, `${this.baseUrl}/api/v2/agents`, params, null, cb)
+	},
+
+	updateAgent(id, data, cb) {
+		makeRequest('PUT', this._auth, `${this.baseUrl}/api/v2/agents/${id}`, null, data, cb)
+	},
+
+	deleteAgent(id, cb) {
+		makeRequest('DELETE', this._auth, `${this.baseUrl}/api/v2/agents/${id}`, null, null, cb)
+	},
+
```

Comment:
Seems like you forgot to implement the remaining one method [currentAgent](https://developers.freshdesk.com/api/#me)